assembl/static2/js/app/components/debate/thread/answerForm.jsx

```diff
@@ -43,9 +47,9 @@ class AnswerForm extends React.PureComponent<*, AnswerFormProps, AnswerFormState
       submitting: false
     };
   }
-
   handleCancel = () => {
-    this.props.hideAnswerForm();
+    const { hideAnswerForm } = this.props;
+    this.setState({ body: null }, hideAnswerForm());
```

Comment:
By doing this, does the previous `draftState` object get garbage collected? Wondering if this could lead to a leak?

Reply:
Sorry, I don't understand what "garbage" means :(

Reply:
@ayyazdaniaryan Why wouldn't it be garbage collected? There is no reference to it, right? The only reference is in state.

Reply:
@paolina92 https://developer.mozilla.org/fr/docs/Web/JavaScript/Gestion_de_la_m%C3%A9moire